client/vnctl/lib/vnctl/cli/datapath.rb

```diff
@@ -10,6 +10,7 @@ class Datapath < Base
       option :is_connected, :type => :boolean, :desc => "Flag that detemines if the datapath is connected or not."
       option :dpid, :type => :string, :desc => "Hexadecimal id for the datapath."
       option :node_id, :type => :string, :desc => "The node id for the datapath."
+      option :enable_ovs_learn_action, :type => :boolean, :desc => "Enables the datapath to use OVS learning flows."
```

Comment:
How about adding `:default => true` here? I know it's already set as such in the database but I think it'll be more clear if set as such here too. Otherwise it gives me the feeling that I have to set this flag to enable it because the flag itself start with the word enable.